aiotg/chat.py

```diff
@@ -8,6 +9,34 @@ class Chat:
     """
     Wrapper for telegram chats, passed to most callbacks
     """
+    async def response(self, regex=r'(.+)', unexpected=None):
+        """
+        Await a message if not match with regex send unexpected message.
+
+        :param str regex: Regex match with message sender
+        :param str unexpected: Message if not match
+        :return Response
+        """
+        self.bot._conversations[self.id] = asyncio.Queue()
+        message = await self.bot._conversations[self.id].get()
+        del self.bot._conversations[self.id]
```

Comment:
If you create Queue only once then perhaps Event is a better option.

And here is the problem, with the "return" in _process_message there is a change to loose some messages.

Reply:
yes Event is a better option, but how to send text messages with queue is easier now could be implemented a list or a dictionary that on one hand has the event and on the other the message but the use of the queue I see it easier you know another way?